### ISSUE TITLE
After system update it doesnt let me login anymore

### ISSUE THREAD

**Issue by sTiKyt** [manilarome/the-glorious-lightdm-webkit2-theme#74]
Throws: 
lightdm_greeter_respond: assertion 'priv->connected' failed every time i press login button after entering password

**Comment by sTiKyt** [comment]
After further research found its like that for any webkit theme, i guess something about it is broken but i couldn't find repository to make an issue

**Comment by redxtech** [comment]
I'm getting the same issue. The only way to log into my computer was to ssh in, and change to the gtk greeter.

I should specify I'm using a different webkit greeter theme, I will check if there's an upstream issue already.

**Comment by frostedpenguin** [comment]
@sTiKyt you are right no webkit theme seems to be working after this update. I followed the link from the aur package and this is the [repo](https://github.com/antergos/web-greeter). It is unfortunately **deprecated** so I guess for the time being the only thing we can do is try to inform the arch wiki to not include it in the list of greeters. And make a pr on this README that the build is failing!

**Comment by sTiKyt** [comment]
Yep, it's deprecated but webkit seems to get updates, meaning there has to be different repo somewhere

**Comment by frostedpenguin** [comment]
@sTiKyt  here is the [log](https://github.com/archlinux/svntogit-community/commits/packages/lightdm-webkit2-greeter/trunk) of the changes made for the package. The 2 last updates at 2022 and 2020 where only on the PKGBUILD file. So I unfortunately  think this greeter is not maintained currently.

**Comment by sTiKyt** [comment]
Oh, then it's dead, has to be moved to AUR i guess

**Comment by frostedpenguin** [comment]
yes I am making the pr here and I will try informing arch wiki in general..
